carl/envs/rna/parse_dot_brackets.py

```diff
@@ -0,0 +1,43 @@
+from pathlib import Path
+
+from typing import List, Optional, Union, Generator
+
+
+def parse_dot_brackets(
```

Comment:
Why are these called dot brackets? What are they?

Reply:
'The most common encoding for the RNA secondary structure is the dot-bracket notation, consisting in a balanced parentheses string composed by a three-character alphabet {.,(,)}, that can be unambiguously converted in the RNA secondary structure.'
That's how the RNA sequences are represented in the data.

Reply:
Done